### ISSUE TITLE
Php would be so nice

### ISSUE THREAD

**Issue by bentxt** [jordwalke/rehp#9]
Hi
I'm aware you might have other plans, but ...

I would be so extremely cool if there was a Php output. Php is so easily installed and deployed. And now that there is rehp all could be done with a good programming language.

Just a thought.
Ben

**Comment by jordwalke** [comment]
Yes, I agree. We currently target Hack which is very very close to PHP, so if someone cared enough they could help us add a flag to the compiler to emit proper PHP - with not much effort.

**Comment by benkb** [comment]
And what would be the prefered form of such a contribution. Forking ?https://guides.github.com/activities/forking/

**Comment by jordwalke** [comment]
@benkb No, the best way would be send PRs to this repo just as you would do for other improvements. We currently have the flag `--backend=js/php`, and I would suggest adding a third value `php-standard`.
